elixir/apps/web/lib/web/live/gateways_live/show.ex

```diff
@@ -70,7 +97,7 @@ defmodule Web.GatewaysLive.Show do
               Location
             </th>
             <td class="px-6 py-4">
-              San Jose, CA
+              <%= @gateway.last_seen_remote_ip %> (TODO: add physical location)
```

Comment:
@jamilbk will we buy MaxMind database?

Reply:
Alternatively, we can just build a link to third-party service with `target="_blank"`, eg.: `https://whatismyipaddress.com/ip/<%= @gateway.last_seen_remote_ip %>`

Reply:
Hm, I believe the GeoLite2 city database is appropriately licensed for our use case. Avoids relying on an external service since I believe we want to capture the location at the time of ingest if we can.

https://dev.maxmind.com/geoip/geolite2-free-geolocation-data

We may need a partnership to avoid violating other services' ToS otherwise.

Reply:
@jamilbk We also will have on-prem users and making it a SQL dump dependency that needs to be maintained is a pain, even for our cloud. Plus resolving location by IP is extremely inaccurate anyways. Maybe then let's just skip it from v1? Admins know how to use IP databases anyways.

Reply:
Well so the justification for this is location-based policies have been a common feature request since we launched. Security teams often use location data to build access policies that fit their compliance requirements.

In general I would prefer not compromising on `cloud` features for the sake of the `onprem` user, since it's unknown if/when we will have any onprem paid customers. We could perhaps architect features like this such that they're easy to deactivate for `onprem` installs until we have the bandwidth to support and document them.

But hm I suppose I agree to saving this as a post-1.0 feature. In that case I would just display the IPv4/6 in perhaps a `<code>` making it easy to copy/paste for the admin to lookup manually.

Reply:
I removed the `TODO: add physical address` and put the IP addr in a `<code>` tag